utils/build-script

```diff
@@ -928,8 +928,8 @@ class BuildScriptInvocation(object):
         # matches that of `build-script-impl`.
         product_classes = self.compute_product_classes()
 
-        impl_product_classes = [cls for cls in product_classes
-                                if cls.is_build_script_impl_product()]
+        standalone_product_classes = [cls for cls in product_classes
+                                      if cls.needs_toolchain()]
```

Comment:
Doesn't this invert the meaning of the loops below?

Reply:
Yes it does 🤷‍♂️ I will have to go back over this because it finishes building correctly in OS X, for some reason.